### PR TITLE
Feature/fsar 107 list sort by

### PR DESCRIPTION
**Pull request by Yinuo-M**
I have implemented the arrow Rachel designed by using CSS "appearance: none" and background properties. This works across browsers, and theoretically "::ms-expanded {display: none}" should do the job for IE11, but I can't test it on IE11 yet.

### CODE REVIEW

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/pattern-library/components/SortBy/sortBy.html.twig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class='sort'>
+	<label for='sort' class='sort__label'>Sort by</label>
+	<select name='sort' id='sort' class='sort__select'>
+		<option value='relevance' class='sort__option' selected>Relevance</option>
+		<option value='recent' class='sort__option'>Most recent</option>
+	</select>
+</div>
```
[the remark this answers is not in the document]
Comment (Yinuo-M) [reply src/pattern-library/components/SortBy/sortBy.html.twig:7]: Fixed

File: src/pattern-library/components/SortBy/arrow-down.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="10" xmlns="http://www.w3.org/2000/svg">
+    <path d="m1.245 1.541 7 7 7-7" stroke="#000" stroke-width="2" fill="none" fill-rule="evenodd"/>
+</svg>
```
[the remark this answers is not in the document]
Comment (Yinuo-M) [reply src/pattern-library/components/SortBy/arrow-down.svg:3]: Fixed

File: src/pattern-library/components/SortBy/sortBy.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    border: 1px solid var(--fsa-primary_black);
+    border-radius: 0;
+
+    /* Add customised arrow */
+    appearance: none;
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    &::ms-expanded {
+      display: none;
+    }
```
[the remark this answers is not in the document]
Comment (Yinuo-M) [reply src/pattern-library/components/SortBy/sortBy.scss:31]: I don't think it has any impact on accessibility. It only affects the appearance. I tried it on several browsers and it works fine with keyboard and works in exactly the same way as gov.uk's dropdown list.

**Comment by Yinuo-M** [comment]
I fixed indents and added texts as parameters. Should be alright now!

**Review by PeterBerryman — APPROVED** [review]

**Pull request merged**